wicket/wicked-charts-wicket8/src/main/java/de/adesso/wickedcharts/wicket8/JavaScriptResourceRegistry.java

```diff
@@ -78,7 +78,7 @@ public static JavaScriptResourceRegistry getInstance() {
         return INSTANCE;
     }
 
-    private RegistryEntry jqueryEntry = new RegistryEntry(JQueryResourceReference.get());
+    private RegistryEntry jqueryEntry = new RegistryEntry(JQueryResourceReference.getV2());
```

Comment:
I agree that this is a good change but it may break apps which use other libs (like the one with KendoUI recently). Is there a way to use whatever is configured by the app in MyApplication#init() with getJavaScriptLibrarySettings().getJQueryReference() ?

Reply:
I believe you can use whichever version you want (V1. V2 or V3) using:
`JavaScriptResourceRegistry.getInstance().setJQueryReference()` in `MyApplication#init()`

Reply:
:+1: